mrmustard/lab_dev/circuit_components.py

```diff
@@ -16,7 +16,7 @@
 A base class for the components of quantum circuits.
 """
 
-# pylint: disable=super-init-not-called, import-outside-toplevel
+# pylint: disable=super-init-not-called, import-outside-toplevel, too-many-positional-arguments
```

Comment:
Out of curiosity which line is raising this pylint issue?

Reply:
pylint started to complain about any method that needs more than 5 args, and it counts also `self` as one, so the init raises this issue